### PR TITLE
Add a muxed account doc to muxed account page

### PR DESCRIPTION
**Pull request by jeesunikim**
- Added by placing an icon next to the title and in the description box below

create-muxed
parse-muxed

@sdfcharles : I noticed that in some areas, we use a book icon for the external link (like in build transaction), for the info icon in the title, should I use `info icon` or `book icon`

- what we have on `build tx`
build-tx

- muxed account title
title-muxed

### CODE REVIEW

**Review by leighmcculloch — APPROVED** [review]

File: src/app/(sidebar)/account/muxed-create/page.tsx [hunk 1]
```diff
@@ -165,6 +168,8 @@ export default function CreateMuxedAccount() {
         title="Muxed accounts are uncommon"
       >
         Don’t use in a production environment unless you know what you’re doing.
+        Read more about Muxed accounts{" "}
+        <Link href="https://developers.stellar.org/">here</Link>.
```
Comment (quietbits) [inline src/app/(sidebar)/account/muxed-create/page.tsx:172]: Should we use a more specific link here (same on the parsed muxed account page)?

File: src/components/WithInfoText/index.tsx [hunk 2]
```diff
@@ -34,7 +34,7 @@ export const WithInfoText = ({
         {children}
 
         <NextLink {...buttonBaseProps} href={href}>
-          <Icon.InfoCircle />
+          <Icon.BookOpen01 />
```
Comment (quietbits) [inline src/components/WithInfoText/index.tsx:37]: If we change the icon here, we should also change the component name.

File: src/app/(sidebar)/account/muxed-create/page.tsx (hunk 1 above)
Comment (janewang) [reply src/app/(sidebar)/account/muxed-create/page.tsx:172]: +1. Maybe this link https://developers.stellar.org/docs/learn/encyclopedia/transactions-specialized/pooled-accounts-muxed-accounts-memos#muxed-accounts

File: src/components/WithInfoText/index.tsx (hunk 2 above)
Comment (jeesunikim) [reply src/components/WithInfoText/index.tsx:37]: i'll keep the original

**Pull request merged**